### PR TITLE
added a podspec for cocoapods

### PR DESCRIPTION
**Pull request by sdgandhi**

### CODE REVIEW

File: IIIAsync.podspec [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+  s.name         = "IIIAsync"
+  s.version      = "1.0.4"
+  s.summary      = "Concurrency control flow system for Objective-C"
+
+  s.description  = <<-DESC
+                   IIIAsync is a control flow system for managing multiple operations serially or in parallel. It can collect the results of multiple operations, either objects or errors. When all operations finish, you get a completion handler with the results in a structured order, regardless of which order they complete. You can run these operations either on the main queue, on a single background thread pool, or on a multi-threaded background pool. It makes it easier to organize code and obtain results, letting you build powerful higher-order operations that rely on multiple asynchronous calls. IIIAsync is heavily modeled after and inspired by the brilliant async.js library by Caolan McMahon.
+                   DESC
+
+  s.homepage     = "https://github.com/sdgandhi/IIIAsync"
```
Comment (jhurliman) [inline IIIAsync.podspec]: Wrong homepage

File: IIIAsync.podspec [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+
+  # ―――  Spec License  ――――――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
+  #
+  #  Licensing your code is important. See http://choosealicense.com for more info.
+  #  CocoaPods will detect a license file if there is a named LICENSE*
+  #  Popular ones are 'MIT', 'BSD' and 'Apache License, Version 2.0'.
+  #
+
+  s.license      = { :type => "As-Is", :file => "LICENSE.md" }
```
Comment (jhurliman) [inline IIIAsync.podspec]: Project is using MIT license

File: IIIAsync.podspec [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+  s.author    = "Steve Streza"
+
+  # ――― Platform Specifics ――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
+  #
+  #  If this Pod runs only on iOS or OS X, then specify the platform and
+  #  the deployment target. You can optionally include the target after the platform.
+  #
+
+  s.platform     = :ios, '7.0'
```
Comment (jhurliman) [inline IIIAsync.podspec]: I believe it is also compatible with 6.0

File: IIIAsync.podspec [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+
+  # ――― Source Location ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
+  #
+  #  Specify the location from where the source should be retrieved.
+  #  Supports git, hg, bzr, svn and HTTP.
+  #
+
+  s.source       = { :git => "https://github.com/sdgandhi/IIIAsync.git", :tag => "1.0.4" }
```
Comment (jhurliman) [inline IIIAsync.podspec]: Wrong source

**Comment by sdgandhi** [comment]
@stevestreza @jhurliman fixed podspec

**Pull request merged**